lib/index.js

```diff
@@ -34,11 +36,10 @@ try {
 try {
   settings = {
     ...settings,
-    ...JSON.parse(fs.readFileSync(settingsFile))
-  };
-} catch (e) {
+    ...yaml.load(fs.readFile(settingsFile, 'utf8'))
+};
```

Comment:
Lines where brackets end should have the same indentation that the line has where they begin.
```js
  // Incorrect
  settings = {
}

  // Correct
  settings = {
  }
```

```suggestion
  };
```

Reply:
followed this, do remined me if it is left anywhere.